tests/kernel/workq/work_queue_api/src/main.c

```diff
@@ -832,20 +944,49 @@ static void work_handler_resubmit(struct k_work *w)
 }
 
 /**
- * @brief Test work submission to queue from handler context, resubmitting
- * a work item during execution of its callback
+ * @brief Test resubmitting work item in callback
  * @details
- * - That test uses sync_sema semaphore with initial count 0.
- * - That test verifies that it is possible during execution of the handler
```

Comment:
These two sentence looks not like "test procedures", how do you think remove "The test" here?